### PR TITLE
Update Solr known limitations

### PR DESCRIPTION
**Pull request by danreb**
This will update Pantheon Solr offering's known limitation to avoid confusing customers

Closes #

## Summary

**[Doc Page Title](https://docs.pantheon.io/doc-title)** - <Enter a one sentence summation of the pertinent changes (including not-yet-completed work) provided by this PR.>

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by danreb** [comment]
The changes made from the PR have been discussed in the PRT search internal Slack channel here https://pantheon.slack.com/archives/C03PBL22K44/p1710891863296069

**Comment by stevector** [comment]
@stovak or @greg-1-anderson can you review to confirm the accuracy of these changes?

File: source/content/solr.md [hunk 1]
```diff
@@ -46,10 +46,10 @@ For installation instructions and additional details, refer to [Enabling Solr fo
 
 - Solr can only be enabled by [Admin users or Users in Charge](/guides/account-mgmt/workspace-sites-teams/teams#site-level-roles-and-permissions).
 
-- Anything that takes more than 5 seconds to send to the Solr server to be indexed will timeout, which will block indexing. For example, large documents attached to Drupal nodes. In these cases, the developer must work with the content or code to exempt the nodes and/or files from being indexed. [Check Index and Batch sizes](#check-index-and-batch-sizes) for possible scenarios or more time-outs.
+- Anything that takes more than 5 seconds to send to the Solr server to be indexed will timeout, which will block indexing. For example, large documents are attached to Drupal nodes. In these cases, the developer must work with the content or code to exempt the nodes and/or files from being indexed. [Check Index and Batch sizes](#check-index-and-batch-sizes) for possible scenarios or more time-outs.
```
Comment (namespacebrian) [inline source/content/solr.md:49]:
```suggestion
- Anything that takes more than 5 seconds to send to the Solr server to be indexed will timeout, which will block indexing. For example, Drupal nodes with large document attached. In these cases, the developer must work with the content or code to exempt the nodes and/or files from being indexed. [Check Index and Batch sizes](#check-index-and-batch-sizes) for possible scenarios or more time-outs.
```

I think the idea here is that a Drupal node might not get indexed if there is a large document file attached to it.

Comment (namespacebrian) [reply source/content/solr.md:49]: That's how this sentence read to me prior to the addition of "are" -- that nodes with large documents attached to them will block indexing.

Comment (stovak) [reply source/content/solr.md:49]: Agree With Brian's suggestion. Approved with those changes.

**Review by stovak — APPROVED** [review]

**Review by namespacebrian — APPROVED** [review]

File: source/content/solr.md [hunk 2]
```diff
@@ -46,10 +46,10 @@ For installation instructions and additional details, refer to [Enabling Solr fo
...
 
-- Anything that takes more than 5 seconds to send to the Solr server to be indexed will timeout, which will block indexing. For example, large documents attached to Drupal nodes. In these cases, the developer must work with the content or code to exempt the nodes and/or files from being indexed. [Check Index and Batch sizes](#check-index-and-batch-sizes) for possible scenarios or more time-outs.
+- Anything that takes more than 5 seconds to send to the Solr server to be indexed will timeout, which will block indexing. For example, Drupal nodes with large document attached. In these cases, the developer must work with the content or code to exempt the nodes and/or files from being indexed. [Check Index and Batch sizes](#check-index-and-batch-sizes) for possible scenarios or more time-outs.
 - If you have a very large number of facets, the URLs generated by CURL may exceed length limits (may be fixed in search\_api).
 
-- Partial search, stopwords.txt, synonyms.txt, elevate.xml, etc. are not supported.
+- Partial search, stopwords.txt, synonyms.txt, elevate.xml, etc. are features currently not supported in Solr version 3 at Pantheon.
```
Comment (stevector) [inline source/content/solr.md:52]:
```suggestion
- Partial search, stopwords.txt, synonyms.txt, elevate.xml, etc. are features not supported in Solr version 3 at Pantheon.
```

**Review by stevector — APPROVED** [review]
Thanks for the reviews @

**Pull request merged**